Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor
+{
+    internal class BlenderLauncher : IDCCLauncher
+    {
+        public const string FileFormat = ".blend";
+
+        const string editorSettingPath = "MESHSYNC_BLENDER_PATH";
+
+        System.Diagnostics.Process m_blenderProcess;
+        static bool redirectBlenderToUnityConsole;
+
+        public RunMode runMode { get; set; }
+
+        ~BlenderLauncher()
+        {
+            Dispose();
+        }
+
+        static string GetBlenderPath()
+        {
+            var blenderPath = EditorPrefs.GetString(editorSettingPath);
+
+            if (string.IsNullOrEmpty(blenderPath))
+            {
+                if (!DefaultAppUtility.TryGetRegisteredApplication(BlenderLauncher.FileFormat, out blenderPath))
+                {
+                    if (Application.platform == RuntimePlatform.OSXEditor)
+                    {
+                        blenderPath = Path.Combine(new[] { "Applications", "Blender.app", "Contents", "MacOS", "Blender" });
```

Comment:
You don't need to create an array, you can just write 
Path.Combine with as many strings as you want as parameters

Reply:
'params' still creates an array but done anyway.